packages/client/mutations/SendClientSideEvent.ts

```diff
@@ -1,22 +1,27 @@
 import graphql from 'babel-plugin-relay/macro'
 import {getRequest} from 'relay-runtime'
 import Atmosphere from '../Atmosphere'
-import {SegmentEventTrackOptions} from '../__generated__/SendClientSegmentEventMutation.graphql'
+import {SegmentEventTrackOptions} from '../__generated__/SendClientSideEventMutation.graphql'
+import * as amplitude from '@amplitude/analytics-browser'
 
 const mutation = graphql`
-  mutation SendClientSegmentEventMutation($event: String!, $options: SegmentEventTrackOptions) {
+  mutation SendClientSideEventMutation($event: String!, $options: SegmentEventTrackOptions) {
     segmentEventTrack(event: $event, options: $options)
   }
 `
 
 export type SendClientSegmentEventOptions = SegmentEventTrackOptions
 
-const SendClientSegmentEventMutation = (
+const SendClientSideEvent = (
   atmosphere: Atmosphere,
   event: string,
   options?: SendClientSegmentEventOptions
 ) => {
   atmosphere.handleFetchPromise(getRequest(mutation).params, {event, options})
```

Comment:
@nickoferrall I'm planning to remove this part and also Line 7 to Line 11, i.e., the mutation, in the very next PR